backend/app/resources/create_invited_user_dto.py

```diff
@@ -0,0 +1,18 @@
+class CreateInvitedUserDTO:
+    def __init__(self, **kwargs):
+        self.email = kwargs.get("email")
+        self.role = kwargs.get("role")
+        self.first_name = kwargs.get("first_name")
+        self.last_name = kwargs.get("last_name")
+
+    def validate(self):
+        error_list = []
+        if type(self.email) is not str:
+            error_list.append("The email supplied is not a string.")
+        if self.role not in ["Admin", "Relief Staff", "Regular Staff"]:
```

Comment:
If we implement the above enum (and import it here), we can change this to be:

```suggestion
        if self.role not in [role.value for role in RoleOptions]
```

Or if we wanna be extra cringe hyper-runtime-efficient leetcode demons 😈 then we could implement it this way too:
```suggestion
        if self.role not in set(role.value for role in RoleOptions)
```